### ISSUE TITLE
`rubocop -a` breaks the following program

### ISSUE THREAD

**Issue by dcu** [bbatsov/rubocop#1879]
The following example is broken after running `rubocop -a`:

```ruby
def foo
  bar 1, { 'a' => 1, 'b' => 2, }
end

def bar(a,b)
  puts a.inspect
  puts b.inspect
end

foo
```

it's converted to this:

```ruby
def foo
  bar 1, 'a' => 1, 'b' => 2,
end

def bar(a,b)
  puts a.inspect
  puts b.inspect
end

foo
```

**Comment by l8nite** [comment]
Ran into this today too. Looks like we need to omit the trailing comma when removing redundant braces.

**Comment by jonas054** [comment]
This has been fixed in #1881 but I made a mistake in the reference to this issue, which is why it's still open. Closing now.
